pkg/template/processor_test.go

```diff
@@ -340,6 +344,27 @@ func TestProcessAndApply(t *testing.T) {
 		assert.Equal(t, "extraUser", binding.Subjects[1].Name)
 	})
 
+	t.Run("should not create or update existing namespace and role binding", func(t *testing.T) {
+		// given
+		cl := NewFakeClient(t)
+		p := template.NewProcessor(cl, s)
+		tmpl, err := DecodeTemplate(decoder,
+			CreateTemplate(WithObjects(Namespace, RoleBinding), WithParams(UsernameParam, CommitParam)))
+		require.NoError(t, err)
+		objs, err := p.Process(tmpl, values)
+		require.NoError(t, err)
+		created, err := p.Apply(objs)
+		require.NoError(t, err)
+		assert.True(t, created)
+		assertNamespaceExists(t, cl, user)
+		assertRoleBindingExists(t, cl, user)
+
+		// when apply the same template again
+		updated, err := p.Apply(objs)
+		require.NoError(t, err)
```

Comment:
please, add the `// then` comment
```suggestion

		// then
		require.NoError(t, err)
```

Reply:
ah yes! thanks!